lib/aca_entities/crm/contracts/account_contract.rb

```diff
@@ -28,15 +28,15 @@ class AccountContract < Dry::Validation::Contract
         params do
           required(:hbxid_c).filled(Types::Coercible::String)
           required(:name).filled(Types::Coercible::String)
-          required(:email1).filled(AcaEntities::Crm::Types::Email)
+          optional(:email1).maybe(AcaEntities::Crm::Types::Email)
```

Comment:
@jacobkagon Can you also update the Entity? Also, tests should be added that check both the contract and entities in the entity spec.